Channel/SwiftmailerChannel.php

```diff
@@ -49,7 +49,7 @@ public function __construct(
      */
     public function supports($recipient)
     {
-        if (is_object($recipient) && $recipient instanceof SwiftmailerRecipientInterface) {
+        if (is_object($recipient) && $recipient instanceof EmailRecipientInterface) {
```

Comment:
Idem `return $recipient instanceof EmailRecipientInterface;`

Reply:
I will simplify this, but I'll keep the `is_object` test. Tried to remove it once, it was less clear.